source/lib/predicates/string.ts

```diff
@@ -156,6 +156,16 @@ export class StringPredicate extends Predicate<string> {
 		});
 	}
 
+	/**
+	 * Test a string to be alphabetical.
+	 */
+	get alphabetical() {
+		return this.addValidator({
+			message: (value, label) => `Expected ${label} to be alphabetical, got \`${value}\``,
+			validator: value => /^[a-z]+$/ig.test(value)
```

Comment:
Should a zero-length string fail this check? seeing as `lowercase` and `uppercase` both succeed for empty strings, I think `alphabetical` should succeed as well.

If we agree on this, then the only change is switching the `+` to a `*`

Reply:
Think it should fail on a zero length string, although it might be annoying. If we make the switch, you could write `nonEmpty.alphabetical` to get the same check. Still it feels weird that it would pass though. What do you guys think?

Reply:
I think it should fail on a zero-length string. We should do the same in the uppercase/lowercase predicates. This is how the `numeric` predicate works already.

Reply:
So I should rewrite the `uppercase` & `lowercase` predicates to fail on zero-length and keep `alphabetical` as it is?

Reply:
Yes